scripts/data_processing/PPI/basemap_mvp.R

```diff
@@ -0,0 +1,300 @@
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "05", "CANTABRIA", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "06", "CASTILLA - LA MANCHA", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "08", "CATALUNA", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "09", "CEUTA", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "10", "EXTREMADURA", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "11", "GALICIA", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "12", "LA RIOJA", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "14", "MELILLA", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "15", "MURCIA", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "16", "NAVARRA", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "17", "PAIS VASCO", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "18", "ASTURIAS", macr_nm)) %>% 
+               mutate(macr_nm = if_else(iso3 == "ESP" & macr_cd == "19", "COMUNITAT VALENCIANA", macr_nm))
+
+##### GBR- at level 3 (but treat as level 2), level 1 in data (first letter of micr_cd)
+#### Assign level 1 names
+Geo.Pop_subdivs <- mutate(Geo.Pop_subdivs, macr_nm = case_when(iso3 == "GBR" & grepl("E", micr_cd) ~ "England", 
+                                                               iso3 == "GBR" & grepl("N", micr_cd) ~ "Northern Ireland", 
+                                                               iso3 == "GBR" & grepl("S", micr_cd) ~ "Scotland", 
+                                                               iso3 == "GBR" & grepl("W", micr_cd) ~ "Wales", 
+                                                               iso3 != "GBR" ~ macr_nm))
+
+##### IND- only 1st level
+
+##### ITA- pop at level 2, level 1 in the data
+
+##### JPN- at level 2, level 1 not in the data
+### Run script to extract raw data for japan- level 1-level 2 keys
+source("japan_rawdata.R")
+### Get get level 1 names in the data
+Geo.Pop_subdivs <- JPN_reg_pref %>% 
+    mutate(iso3 = "JPN", micr_nm = str_to_upper(prefecture)) %>% 
+    select(-prefecture) %>%
+    left_join(Geo.Pop_subdivs, .) %>% 
+    mutate(macr_nm = case_when(iso3 == "JPN" ~ region, 
+                               iso3 != "JPN" ~ macr_nm)) %>% 
+    select(-region)
+
+## Free up memory
+rm(Japanregions, JPNprefectures, dataJapan, JPN_reg_pref)
+gc()
+
+##### MYS- only 1st level
+
+##### NLD- pop at level 2, level 1 in the data
+
+##### PHL- pop at level 2, level 1 in the data
+
+##### SAU- only 1st level
+
+##### SWE- only 1st level
+
+##### THA- pop at level 2, level 1 in the data
+
+##### TWN- only 1st level
+
+##### USA- pop at level 2, level 1 in the data
```

Comment:
Question, are these placeholders to do the aggregation?? I.e. in the Beckett data we have population by country, and want to aggregate to state as well?  What do you mean when you say level 1 in the data?